src/main/java/com/yanirta/ImageTester.java

```diff
@@ -20,9 +20,10 @@
 import java.io.IOException;
 import java.security.KeyManagementException;
 import java.security.NoSuchAlgorithmException;
+import java.util.UUID;
```

Comment:
Is this being used?

Reply:
For now no, I will use it later to generate a batchId
Removing for this version